src/workerd/api/http.h

```diff
@@ -384,6 +384,14 @@ class Fetcher: public jsg::Object {
     virtual kj::Own<WorkerInterface> newSingleUseClient(kj::Maybe<kj::String> cfStr) = 0;
   };
 
+  class CrossContextOutgoingFactory {
+    // Used by Fetchers that obtain their HttpClient in a custom way, but which aren't tied
```

Comment:
Nit: this "in a custom way" could use a bit of clarification here. Maybe point to or describe an example?

Reply:
It just means as opposed to simply using a numbered subrequest channel...

Reply:
Yes, I know that... someone else coming into the code later may not and a bit more detail in the comment would be helpful.

Reply:
The meaning seems clear enough to me and not worth all the rebasing and re-running of CI needed to change it. Up to @ObsidianMinor if he wants to change, otherwise I'm happy to merge this as-is.

Reply:
Yeah it seems obvious to me too with the naming. It's just OutgoingFactory (like for actors), but it's not tied to an IO context.